packages/relay/src/lib/clients/cache/redisCache.ts

```diff
@@ -39,7 +40,8 @@ export class RedisCache implements IRedisCacheClient {
    */
   private readonly options = {
     // Max time to live in ms, for items before they are considered stale.
-    ttl: Number.parseInt(process.env.CACHE_TTL ?? constants.CACHE_TTL.ONE_HOUR.toString()),
+    // @ts-ignore
```

Comment:
Q: I noticed `@ts-ignore` is added wherever `new ConfigService` is used, and there are so many of them now. I think we should fix this issue so we don’t have to ignore the execution every time we get an ENV var.

Reply:
You're right. There are unnecessary transformations like this above. The `process.env.CACHE_TTL` will be parsed to the number in the config service and here we cast it to string (via `toString()`) and then again cast it to the number. All these places must be fixed in another issue/PR, we wanted to stick to as few as possible new changes that can break something that already works.